docs/source/es/quicktour.mdx

```diff
@@ -80,24 +80,24 @@ Importa [`pipeline`] y especifica la tarea que deseas completar:
 ```py
 >>> from transformers import pipeline
 
->>> classifier = pipeline("sentiment-analysis")
+>>> clasificador = pipeline('sentiment-analysis', model="pysentimiento/robertuito-sentiment-analysis")
 ```
 
-El pipeline descarga y almacena en caché un [modelo preentrenado](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) por defecto y tokeniza para análisis de sentimiento. Ahora puedes usar `classifier` en tu texto objetivo:
+El pipeline descarga y almacena en caché el [modelo preentrenado](https://huggingface.co/pysentimiento/robertuito-sentiment-analysis) y tokeniza para análisis de sentimiento. Si no hubieramos elegido un modelo el pipeline habría elegido uno por defecto. Ahora puedes usar `clasificador` en tu texto objetivo:
 
 ```py
->>> classifier("We are very happy to show you the 🤗 Transformers library.")
-[{'label': 'POSITIVE', 'score': 0.9998}]
+>>> clasificador("Estamos muy felices de mostrarte la biblioteca de 🤗 Transformers.")
+[{'label': 'POS', 'score': 0.9916}]
 ```
 
-Para más de un enunciado entrega una lista de frases al [`pipeline`] que devolverá una lista de diccionarios:
+Para más de un enunciado, entrega una lista al [`pipeline`] que devolverá una lista de diccionarios:
 
 ```py
->>> results = classifier(["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."])
->>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
-label: POSITIVE, with score: 0.9998
-label: NEGATIVE, with score: 0.5309
+>>> resultados = clasificador(["Estamos muy felices de mostrarte la biblioteca de 🤗 Transformers.", ""Esperamos que odies lo mucho que te gusta 😈."])
```

Comment:
Looks like you have an extra double quote here:  `""Esperamos`  because of which build is failing.

Reply:
Thank you @kumar-abhishek! That was the error! :)

Reply:
@omarespejel there is still a build failure, please fix that.